src/main/kotlin/io/streammachine/api/cli/commands/KafkaExportersUsers.kt

```diff
@@ -0,0 +1,68 @@
+package io.streammachine.api.cli.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.kittinunf.fuel.httpDelete
+import com.github.kittinunf.fuel.httpGet
+import com.github.kittinunf.fuel.httpPost
+import io.streammachine.api.cli.common.printResponse
+
+class KafkaExportersUsers : CliktCommand(
+    name = "kafka-exporters-users",
+    help = "View users on kafka-exporters and their details."
+) {
+    init {
+        subcommands(
+            kafkaExportersUsersList(),
+            KafkaExportersUsersGetOne(),
+            KafkaExportersUsersCreate(),
+            KafkaExportersUsersDelete()
+        )
+    }
+
+    override fun run() = Unit
+}
+
+open class kafkaExportersUsersList : CliktCommand(
+    name = "list",
+    help = "get all users on a certain Kafka exporter"
+) {
+    internal val streamName by argument("stream-name", help = "Name of the stream that is being exporter")
+
+    override fun run() {
+        "/v1/kafka-exporters-users/$streamName".httpGet().printResponse()
+    }
+}
+open class KafkaExportersUsersGetOne : CliktCommand(
+    name = "get",
+    help = "Get a specific user on a Kafka exporter by stream name"
+) {
+    internal val streamName by argument("stream-name", help = "Name of the stream that is being exported")
+    internal val clientId by argument("client-id", help = "The client/consumer id that identifies the user")
```

Comment:
How come this is a `consumer id`? It's the client id for this stream, that the customer uses to authenticate their app with Stream Machine in order to start consuming data, right? Hence, it's a client credentials flow in OAuth terms.